{
    "total_results": 3,
    "messages": [
        [
            {
                "id": "920322788441604137",
                "type": 0,
                "content": "Hi where do i find kevinheavey ?",
                "channel_id": "899315715381796904",
                "author": {
                    "id": "456226577798135808",
                    "username": "Deleted User",
                    "avatar": null,
                    "discriminator": "0000",
                    "public_flags": 0
                },
                "attachments": [],
                "embeds": [],
                "mentions": [],
                "mention_roles": [],
                "pinned": false,
                "mention_everyone": false,
                "tts": false,
                "timestamp": "2021-12-14T14:34:11.535000+00:00",
                "edited_timestamp": null,
                "flags": 0,
                "components": [],
                "hit": true
            }
        ],
        [
            {
                "id": "920157108727582761",
                "type": 0,
                "content": "I\u2019ve seen a similar message when passing the incorrect number of arguments or accounts to an RPC call, fwiw",
                "channel_id": "899315715381796904",
                "author": {
                    "id": "886273867293802556",
                    "username": "joebuild",
                    "avatar": "8bd105a959e9fb8265f56a57776b3f6a",
                    "discriminator": "8170",
                    "public_flags": 0
                },
                "attachments": [],
                "embeds": [],
                "mentions": [],
                "mention_roles": [],
                "pinned": false,
                "mention_everyone": false,
                "tts": false,
                "timestamp": "2021-12-14T03:35:50.413000+00:00",
                "edited_timestamp": null,
                "flags": 0,
                "components": [],
                "hit": true
            }
        ],
        [
            {
                "id": "920154391040901210",
                "type": 0,
                "content": "anyone see this RPCException before? what is the index out of bounds with?\n\n`RPCException: {'code': -32602, 'message': 'invalid transaction: index out of bounds'}`",
                "channel_id": "899315715381796904",
                "author": {
                    "id": "839599169915846697",
                    "username": "bigz_Pubkey",
                    "avatar": "90c04e0377712fa10aba17dc2dd06f45",
                    "discriminator": "9718",
                    "public_flags": 0
                },
                "attachments": [],
                "embeds": [],
                "mentions": [],
                "mention_roles": [],
                "pinned": false,
                "mention_everyone": false,
                "tts": false,
                "timestamp": "2021-12-14T03:25:02.466000+00:00",
                "edited_timestamp": "2021-12-14T03:26:25.504000+00:00",
                "flags": 0,
                "components": [],
                "hit": true
            }
        ]
    ],
    "analytics_id": "8001f50732a87d49c27cd9c274a2837e"
}